### ISSUE TITLE
Hours are rolling backwards

### ISSUE THREAD

**Issue by LotyM1** [gibbed/SteamAchievementManager#461]
If I want to use a program to increase the hours in the stats, they are often almost completely rolled back after closing the game. example Doki Doki Doki Literature Club. before closing it showed 11 hours. as soon as the game was closed via Steam, it was already 48 minutes

**Comment by dochankpin** [comment]
The same thing happened to me.
And it's not just in one specific game.
But in several games, and randomly.
Example:
I opened several games at the same time (To farm several cards at the same time).
Some of these games were: Resident Evil 2 Remake, Metal Gear V The Phantom Pain, and The Witcher 3 Wild Hunt.
I left the computer on for almost a whole day to farm the cards.
Before that, each game had a total of 30 minutes played (in hours of play on Steam).
So after spending almost a whole day on it, I went to look at the number of hours and it was already at about 19:40h (Each game mentioned above).
So I decided to close the SAM windows one by one.
When I went to analyze each game above, only the exact 30 minutes played were listed (Before using SAM).
In short, it's as if I hadn't even used SAM to farm hours.
